nodes/mtdevice.py

```diff
@@ -1699,6 +1811,46 @@ def get_settings(arg):
     settings = timestamp | orient_mode | calib_mode | NED
     return settings
 
+def get_synchronization_settings(arg):
+    """Parse command line synchronization-settings argument."""
+    # Parse each field from the argument
+    settings = arg.split(',')
+    settings = tuple([int(i) for i in settings]) 
```

Comment:
It'd be better if you could protect this `int(i)` call with a try/except block (and a suitable error message).